stdlib/time/src/main/ballerina/time/timelib.bal

```diff
@@ -48,7 +56,7 @@ public extern function toString(Time time) returns string;
 # + time - The Time record to be formatted
 # + timeFormat - The format which is used to format the time represented by this object
 # + return - The formatted string of the given time
-public extern function format(Time time, string|TimeFormat timeFormat) returns string;
+public extern function format(Time time, string|TimeFormat timeFormat) returns string|error;
```

Comment:
Don't we need to specify in the return statement that this will return an error?